torchani/data/cache_aev.py

```diff
@@ -1,9 +1,10 @@
 # -*- coding: utf-8 -*-
-"""AEVs for a dataset can be precomputed by invoking
-``python -m torchani.data.cache_aev``, this would dump the dataset and
-computed aevs. Use the ``-h`` option for help.
+"""```'torchani.data.cache_aev'``` is deprecated, AEVs for a dataset
+can be precomputed by invoking ``python -m torchani.data.cache_sparse_aev``,
```

Comment:
Could you also add a `cache_sparse_aev.py` that replicate this file? Otherwise `python -m torchani.data.cache_sparse_aev` will not work. Also add a test at:
https://github.com/aiqm/torchani/blob/master/azure/runnable_submodules.yml#L24